### PR TITLE
[istio][docs] Improving estimated overhead section

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Improving estimated overhead section in istio module docs.

## Why do we need it, and what problem does it solve?
The old one was quite poor.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: istio
type: chore
summary: Improved estimated overhead docs section.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

File: modules/110-istio/docs/README_RU.md [hunk 1]
```diff
@@ -304,5 +304,27 @@ Istio позволяет осуществлять сбор трейсов с п
...
+Примерная оценка накладных расходов для экземпляра control-plane, который обслуживает 1000 сервисов и 2000 istio-сайдкаров — 1 vCPU и 1.5GB RAM.
+
+### data-plane
+
+На потребление ресурсов data-plane (istio-сайдкары) влияет множество факторов:
+
+* количество соединений,
+* интенсивность запросов,
+* размер запросов и ответов,
+* протокол (http/tcp),
```
Comment (z9r5) [inline modules/110-istio/docs/README_RU.md:324]:
```suggestion
* протокол (HTTP/TCP),
```

File: modules/110-istio/docs/README.md [hunk 2]
```diff
@@ -301,5 +301,26 @@ To create a multicluster, you need to create a set of `IstioMulticluster` resour
...
+A rough estimate of overhead for a control-plane instance that serves 1000 services and 2000 istio-sidecars is 1 vCPU and 1.5GB RAM.
+
+### data-plane
+
+The consumption of data-plane resources (istio-sidecar) is affected by many factors:
+
+* number of connections,
+* the intensity of requests,
+* size of requests and responses,
+* protocol (http/tcp),
```
Comment (z9r5) [inline modules/110-istio/docs/README.md:321]:
```suggestion
* protocol (HTTP/TCP),
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9544](https://github.com/deckhouse/deckhouse/pull/9544) to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) successful!

**Pull request merged**